ocs_ci/templates/openshift-infra/logging-deployment/clusterlogging_operator/cl-og.yaml

```diff
@@ -1,10 +1,7 @@
 apiVersion: operators.coreos.com/v1
 kind: OperatorGroup
 metadata:
-  generateName: openshift-logging-
+  name: cluster-logging
   namespace: openshift-logging
-  labels:
```

Comment:
I see labels and targetNamespaces being removed. Please check for replacements if any